projects/ngx-datatable/src/lib/components/datatable.component.html

```diff
@@ -66,6 +67,7 @@
     (scroll)="onBodyScroll($event)"
     (treeAction)="onTreeAction($event)"
   >
+    <ng-content select="[loading-indicator]" ngProjectAs="[loading-indicator]"></ng-content>
```

Comment:
This is perfect, I think the table should than check if a value is set and use it internally instead of the linear progress indicator.

Reply:
the problem is that Angular lacks the capability to specify a default content if the select doesn't match. I.e. angular/angular#12530 

There's this workaround, but it's very ugly:

```
           <div #ref><ng-content></ng-content></div> 
           <span *ngIf="ref.childNodes.length == 0">
              Display this if ng-content is empty!
           </span>
```

Reply:
Yes, I did this kind of workaround only since there is not a good way to check whether the select exists or not.